installer/manifests/keptn/charts/control-plane/templates/shipyard-controller.yaml

```diff
@@ -113,16 +113,7 @@ spec:
             - containerPort: 8080
           {{- include "keptn.distributor.resources" . | nindent 10 }}
           env:
-            - name: PUBSUB_URL
-              value: 'nats://keptn-nats-cluster'
-            - name: PUBSUB_TOPIC
-              value: 'sh.keptn.>'
-            - name: PUBSUB_RECIPIENT
-              value: '127.0.0.1'
-            - name: PUBSUB_RECIPIENT_PATH
-              value: '/v1/event'
-            - name: DISABLE_REGISTRATION
-              value: "true"
```

Comment:
Those variables are not all in the `dist.common.env.vars`. Is there no need for those anymore?

Reply:
yes, since the shipyard controller is using the distributor only for sending events with this PR.